### DISCUSSION TITLE
A little rant (NOT about A1111)

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#2611] [General]
a little rant (NOT about A1111) 
I hadn't used SD for a few weeks
and I've just noticed some changes in the interface. Most of the time, they're improvements, but here, in my opinion, they're not...

I find that the system for using and saving styles 
is so badly designed that it takes much longer to move a project forward (now I prefer to copy/paste my positive/negative prompts into a text file than to use the style editor).
 which is really impractical.... (going back and forth in a popup window) 

what do you think?

**Comment by TheLastBen** [comment]
Well, that's out of my hands, the A1111 repo isn't mine
